src/www/css/app.css

```diff
@@ -1113,10 +1154,6 @@ video {
   padding-bottom: 3rem;
 }
 
-.pb-2 {
```

Comment:
Can I just check that these are expected deletions?

Reply:
Same here.